src/styles/includes/_app-styling-variables.scss

```diff
@@ -0,0 +1,47 @@
+// This file should only contain style variables.
+//
+// Never use color variables directly, you should always use a color palette
+// and you must provide a contrast for every defined color.
+// @see http://contrast-ratio.com/
+
+// This mixin is used to contain variables that should not be used globally.
+// Use the `!global` modifier to export them to the global scope.
+@mixin app_define_global_variables {
```

Comment:
You can hide variables from the global SCSS scope by defining them inside a mixin, and just exporting the variables you need by using the `!global` flag.